packages/lib/src/components/internal/Address/components/AddressSearch.tsx

```diff
@@ -33,9 +52,9 @@ export default function AddressSearch({ onAddressLookup, onSelect, onManualAddre
     const mapDataToSelect = data => data.map(({ id, name }) => ({ id, name }));
 
     const onInput = useCallback(
-        async event => {
+        async (inputValue: string) => {
             new Promise<Array<AddressLookupItem>>((resolve, reject) => {
-                onAddressLookup(event, { resolve, reject });
+                onAddressLookup(inputValue, { resolve, reject });
             })
                 .then(data => {
                     setOriginalData(data);
```

Comment:
What is the original data in this case? Is the data formatted according to the `AddressLookupItem` type? or is it just `any` data?

Adding types can help understand the logic better. It can be nice to add the type definitions to the `useState` too if possible, so it is clear what is being stored